app/templates/auth/register.html.j2

```diff
@@ -88,15 +88,16 @@
 
 
     ////////////////////////////////////////////////////////////////////////////////////////////////////
-    // Make sure the entered email address is a vaild email format
+    // Make sure the entered email address is a vaild email format.
+    // emailRegExp source: https://www.w3resource.com/javascript/form/email-validation.php
     ////////////////////////////////////////////////////////////////////////////////////////////////////
     function emailValidate() {
         const email = document.getElementById('email');
+
         const emailRegExp =
-        /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+        /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
I think this should be (I added a + sign as an allowed character):

`^\w+([\.-\\+]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$`

Gmail email addresses allow a plus sign, such as `jdstone1+twitter@gmail.com`.

I think the regex is correct, I tested it here: https://regex101.com